ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+#### Subject of the issue:
+Describe your issue here
+
+#### Your environment:
+* version/branch
+* which browser and its version
```

Comment:
I don't think that there's much of use out of it, we can probably assume that every member is using a "recent enough" one.

Reply:
Ditto, I would leave it as a good practice.